Wikipedia/Code/WMFFeedContentFetcher.m

```diff
@@ -90,6 +90,17 @@ - (void)fetchFeedContentForURL:(NSURL *)siteURL date:(NSDate *)date force:(BOOL)
                                           userInfo:nil]);
 
             } else {
+                NSHTTPURLResponse *response = ((NSHTTPURLResponse *)[operation response]);
+                NSDictionary *headers = [response allHeaderFields];
+                NSString *cacheControllHeader = headers[@"Cache-Control"];
+
+                NSRange range1 = [cacheControllHeader rangeOfString:@"max-age="];
+                NSRange range2 = NSMakeRange(cacheControllHeader.length, 1);
+                NSRange range = NSMakeRange(range1.location + range1.length, range2.location - range1.location - range1.length);
```

Comment:
`range2.length` is never used - can `range2.location` just be `cacheControlHeader.length`?